cv32e40p/tests/embench/pylib/run_corev32.py

```diff
@@ -100,16 +100,11 @@ def build_benchmark_cmd(bench, args):
     global cpu_per 
     cpu_per = float(1/(args.cpu_mhz*1_000_000))
     
-    if args.cfg == 'default':
-      core_config = f'CFG=default'
-    else:
-      core_config = f'CFG=pulp'
-
     #Utilize "make test" environment in core-v-verif
     if args.sim_parallel == 'YES':
-      return ['make', '-C', args.make_path, 'test', f"TEST=emb_{bench}", f"SIMULATOR={args.simulator}", 'USE_ISS=NO', core_config, 'COMP=NO', 'USER_RUN_FLAGS=+rand_stall_obi_disable']
```

Comment:
Why was this flag removed? I do not think the flag should be set in the python script directly but rather in the cfg yaml - but now it is in neither place, would this not cause the introduction of random obi bus stalls?

Reply:
Thank you for the explanation - that should resolve my only concern for this PR